clang-tools-extra/clang-tidy/bugprone/NonZeroEnumToBoolConversionCheck.cpp

```diff
@@ -49,6 +49,9 @@ bool NonZeroEnumToBoolConversionCheck::isLanguageVersionSupported(
 }
 
 void NonZeroEnumToBoolConversionCheck::registerMatchers(MatchFinder *Finder) {
+  auto ExcludedOperators = binaryOperation(hasAnyOperatorName(
+      "|", "&", "^", "<<", ">>", "~", "|=", "&=", "^=", "<<=", ">>="));
```

Comment:
> If you want, then yes, I can reduce this back to overload operators calls only.

I think current implement it fine, but maybe add some comment here to clarify why you want to exclude those operators. Because actually it's for the operator overloading instead of explicit operator.